### PR TITLE
added the menu navigation page

### PR DESCRIPTION
**Pull request by VGH1**
added the menu navigation

### CODE REVIEW

**Comment by VGH1** [comment]
@alexziweiwang @jin-sj please review my code, let me know if there's anything I can fix.

File: generating_pages.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import csv
+import copy
+import random
+from pages import Page
+
+page_list = []
+
+
+with open("word_list.csv") as csv_file:
```
Comment (jin-sj) [inline generating_pages.py]: make into a function

File: generating_pages.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    sample_dict[0] = page_1
+    sample_dict[1] = page_2
+    return sample_dict
+
+
+def generate_easy_page():
+    sample_dict = generate_easy_task()
+    displaying_page = []
+    for i in range(10):
+        PAGE_1 = sample_dict[0]
```
Comment (jin-sj) [inline generating_pages.py]: local variables should be lower case

File: main.py [hunk 3]
```diff
@@ -57,38 +63,50 @@ def connect_to_server(dummy_server=False):
...
+
+
 @app.route('/question/')
 def question_start():
-    send_data('baselinestart')
-    session['uid'] = uuid.uuid1()
-    time.sleep(3)
-    send_data('baselineend')
-    time.sleep(0.5)
+    # send_data('baselinestart')
```
Comment (jin-sj) [inline main.py]: remove commented code

File: main.py [hunk 4]
```diff
@@ -57,38 +63,50 @@ def connect_to_server(dummy_server=False):
...
+        session['error_count'] = {}
+    if page_str not in session['error_count']:
+        session['error_count'][page_str] = 0
+
     # this is called after I press next on the survey
 
-    if canSendMarker and page == 10:
-        server_socket.send_data('hardstart')
-        canSendMarker = False # prevent from sending markers on wrong answers
+    # if canSendMarker and page == 10:
```
Comment (jin-sj) [inline main.py]: rm commented code

File: main.py [hunk 5]
```diff
@@ -57,38 +63,50 @@ def connect_to_server(dummy_server=False):
...
         session.pop('error_count')
         session.pop('uid')
         return render_template("cong.html")
 
     # Adding condition for method!=POST so, we only send marker when page loads,
     # not twice, for when page loads, and when the user submits an answer.
-    if PAGES[page].marker_data!='' and request.method!='POST':
-        server_socket.send_data(PAGES[page].marker_data)
+    # if PAGES[page].marker_data!='' and request.method!='POST':
+    #     server_socket.send_data(PAGES[page].marker_data)
```
Comment (jin-sj) [inline main.py:109]: rm commented code

File: main.py [hunk 6]
```diff
@@ -98,21 +116,20 @@ def question(page):
...
-
-                if page == len(PAGES) - 1: # I finished the hard and will start survey so I will send hardend before starting the survey
-                    send_data('hardend')
+            # if canSendMarker:
+            #     if page == 9: # I finished the easy and will start survey so I will send easyend before starting the survey
+            #         send_data('easyend')
+            #         # I didn't change canSendMarker to false here because I need to send hard start when the new page 10 is loaded
 
+            #     if page == len(PAGES) - 1: # I finished the hard and will start survey so I will send hardend before starting the survey
+            #         send_data('hardend')
```
Comment (jin-sj) [inline main.py]: commented code

File: page_list.py [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+# 1 means easy
+# 2 means difficult
+PATTERN = [1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2]
+
+
+# Create a PAGES list for test page.
+PAGES = []
+
+
+for i in PATTERN:
```
Comment (jin-sj) [inline page_list.py]: make into a function

**Review by jin-sj — CHANGES_REQUESTED** [review]

**Pull request merged**